docs/user/training_a_policy.md

```diff
@@ -0,0 +1,145 @@
+# Training a policy to solve an environment
+This page will guide you training a policy to solve an environment.
+### Define an experiment
+In garage, we train a policy in an experiment, which is a function wrapped by a decorator called `wrap_experiment`. Below is an simple example. `wrap_experiment` could have some arguments. You can see the [experiments doc](https://garage.readthedocs.io/en/latest/user/experiments.html) for details of running experiments.
```

Comment:
i think you need to use a relative link here. you don't know which version of the docs the user is reading.

See https://github.com/readthedocs/recommonmark/issues/108

Reply:
Good idea. Will do.